receiver/jmxreceiver/config.go

```diff
@@ -95,10 +96,21 @@ func (oec otlpExporterConfig) headersToString() string {
 }
 
 func (c *Config) parseProperties() []string {
-	parsed := make([]string, 0, len(c.Properties))
-	for property, value := range c.Properties {
-		parsed = append(parsed, fmt.Sprintf("-D%s=%s", property, value))
+	parsed := make([]string, 0, 1)
+	if len(c.ResourceAttributes) > 0 {
+		attributes := make([]string, 0, len(c.ResourceAttributes))
+		for k, v := range c.ResourceAttributes {
+			attributes = append(attributes, fmt.Sprintf("%s=%s", k, v))
```

Comment:
This appears to be an opportunity to inject additional properties. See comment [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/6750#issuecomment-1104431327).

Reply:
I'll add it into the property file, though it won't be fully supported there until https://github.com/open-telemetry/opentelemetry-java-contrib/pull/323 is merged and released.

Reply:
> I'll add it into the property file

Can you clarify what would be added?

What I was figuring is that we should sanitize (eg remove newlines) from the parameter.

Reply:
Newlines are escaped (see https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/9721/files#diff-82229ece08272c3aec0d722b8c4536e99a33724e47ea1b875c435dea7442e49dR243 ) when writing the values to a properties file. If you'd rather I brought it back into command line properties we can do the same there, or just remove them.

Reply:
Got it. Thanks. Lgtm